odoo/modules/loading.py

```diff
@@ -274,11 +282,20 @@ def load_module_graph(cr, graph, status=None, perform_checks=True,
                     delattr(package, kind)
             module.flush()
 
-        _logger.log(module_log_level, "Module %s loaded in %.2fs, %s queries (+%s extra)",
-                    module_name,
-                    time.time() - module_t0,
-                    cr.sql_log_count - module_cursor_query_count,
-                    odoo.sql_db.sql_counter - module_extra_query_count)  # extra queries: testes, notify, any other closed cursor
+        _logger.log(
+            module_log_level, "Module %s loaded in %.2fs%s, %s queries (+%s extra%s)",
```

Comment:
> The string formating is a little weird, why not keeping test_time and test_queries as numerical values and formatting everything here? 

Because then you have useless noise when *not* running tests. That tests took 0 seconds and 0 queries is of no use when you're not running tests in the first place.

> Also is it intended to keep test queries in extra?

As of then yes, but see discussion with @Elkasitu it could definitely be useful and clearer to split them completely, subtracting the test queries from extras (and possibly also the test time from the module load time? less clear)

Reply:
> it could definitely be useful and clearer to split them completely

Yes I agree, this is why keeping them as numerical value until formating may help. 

> Because then you have useless noise when not running tests. That tests took 0 seconds and 0 queries is of no use when you're not running tests in the first place

There is maybe another way to avoid this noise by conditionally formatting. The string. It's maybe just me but I dont like to have very similar info formatted at different level, inserting a formated test info while formating extra info. I would go for something like that: 
Quick example (not so clean)
```python3
   nb_test_queries = ...
   nb_extra_queries = odoo.sql_db.sql_counter - module_extra_query_count - nb_test_queries
  ...
            module_log_level, "Module %s loaded in %.2fs%s, %s queries%s%s",
   ...
            ' (+%s test)' % nb_test_queries if nb_test_queries else '',
            ' (+%s extra)' % nb_extra_queries if nb_extra_queries else '',
```
By removing extra most of the time we will also remove this noise when this is not usefull since for most module extra queries are only test queries [(267/282 module don't have extra queries without tests) 
](http://runbot35.odoo.com/runbot/static/build/3853554-master/logs/install_all.txt)